docs-content/getting-started/3_client-sdk/7_replay-configuration/proxying-highlight.md

```diff
@@ -17,17 +17,16 @@ One way we can avoid this is by setting up proxy from your domain to Highlight.
 
 ## Setting up the proxy
 
-1.  Upgrade to the Startup Tier [https://www.highlight.io/pricing](https://www.highlight.io/pricing)
+1.  On your domain, add a `CNAME` record that points `highlight.<YOUR_DOMAIN>` to `pub.highlight.run`.
 
-2.  On your domain, add a `CNAME` record that points `highlight.<YOUR_DOMAIN>` to `pub.highlight.run`
+2.  Send us an email at <sales@highlight.io> so we can send over a cost proposal for your annual usage.
```

Comment:
```suggestion
2.  Send us an email at sales@highlight.io so we can send over a cost proposal for your annual usage.
```

Looks like just removing the `<` and `>` should get it to wor and look how we want it
<img width="766" alt="Screenshot 2023-09-06 at 11 15 08 AM" src="https://github.com/highlight/highlight/assets/17744174/e69ba2a3-66b5-45ab-b089-feec9e6eb541">